apps/music/js/ui/views/list_view.js

```diff
@@ -75,9 +75,8 @@ var ListView = {
 
   hideSearch: function lv_hideSearch() {
     this.searchInput.value = '';
-    // XXX: we probably want to animate this...
-    if (this.view.scrollTop < this.searchBox.offsetHeight) {
-      this.view.scrollTop = this.searchBox.offsetHeight;
+    if (this.view.scrollTop < this.anchor.offsetTop) {
+      this.anchor.scrollIntoView({behavior: 'smooth'});
```

Comment:
@jimporter This is really COOL! I don't know we have this api and can make the transition smooth.